docs/artifacthub-repo.yml

```diff
@@ -0,0 +1,6 @@
+# Configuration for our Artifact Hub repository
+# See https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-repo.yml
```

Comment:
Care with this line because it still hits master

Reply:
That's artifact hub, that's OK.